src/ontology/mondo.Makefile

```diff
@@ -264,7 +264,37 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
-	
+#############################################
+##### One-time scripts ######################
+#############################################
+# MedGen conflicts update
+## - main pipeline
+tmp/July2023_CUIReports_FromMedGentoMondo.xlsx:
+	mkdir -p tmp && wget "https://github.com/monarch-initiative/mondo-ingest/files/12029712/July2023_CUIReports_FromMedGentoMondo.xlsx" -O $@
+
+tmp/bad-medgen-xrefs.txt: tmp/July2023_CUIReports_FromMedGentoMondo.xlsx
+	python ../scripts/bad_medgen_xrefs_collate.py -i tmp/July2023_CUIReports_FromMedGentoMondo.xlsx -o $@
```

Comment:
- Consider using this for find/replace https://github.com/monarch-initiative/mondo/blob/master/src/scripts/clean-british-english.py
- Delete all `Medgen:` refs. can do manually

Reply:
Nico will handle these 6 cases:
```
non_xref_removals_content     "synonym: ""Ullrich disease"" RELATED [DOID:0050558, https://www.ncbi.nlm.nih.gov/gtr/conditions/C0410179/]
relationship: excluded_subClassOf MONDO:0016697 {source=""UMLS:CN201941""} ! low grade ependymoma
comment: Other ontologies classify peripheral neuroepithelioma as a seprate class but NCIT treats these as synonyms and we aim to follow NCIT here. (See HP:0006717, SNOMED:254764001, the ""included"" disease concept on OMIM:612219, and UMLS:C3489398).
synonym: ""polysyndactyly"" EXACT [SCTID:84598000, UMLS:C0265553]
synonym: ""thrombophilia, hereditary"" EXACT [MESH:C540694, UMLS:C2584620]
synonym: ""triple positive breast cancer"" EXACT [UMLS:C5243562]
```